.github/workflows/trivy_scan_before_release.yml

```diff
@@ -0,0 +1,49 @@
+name: Scan Theia Docker image for vulnerabilities before release
+
+on:
+  pull_request:
+    branches:
+      - release-*
+
+jobs:
+  build:
+    if: startsWith(github.event.pull_request.title, 'Release ')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - spark-jobs
+          - clickhouse-monitor
+          - clickhouse-server
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build theia Docker images
+      id: make
+      run: |
+        make ${{ matrix.image }}
+    - name: Run Trivy vulnerability scanner on latest Theia ${{ matrix.image }} image
+      if: ${{ always() && steps.make.conclusion == 'success' }}
```

Comment:
it feels like this if condition doesn't serve much of a purpose? `always()` is used to make sure that the job always runs, even if previous steps failed. And you are adding a condition to only run the job is the previous step is successful. Let me know if I'm missing something.

Reply:
In order to fetch the latest image, we should be relying for the make to pass in the prev step.

Reply:
We are not fetching the image, I think we are building it locally?
My point is that if you omit this one line altogether, it should have the same effect (i.e., skip this step if the previous one failed).

Reply:
oh ok, yes that make sense